components/FilterBar.js

```diff
@@ -1,44 +1,68 @@
 import { useState } from "react";
 import useFilters from "@/hooks/useFilters";
-
-export default function FilterBar({
-  onChange,
-  groupedCityEvents,
-  groupedCategoryEvents,
-  filters,
-  reset,
-}) {
+import styled from "styled-components";
+import Button from "./Button";
+import useSWR from "swr";
+const StyledDiv = styled.div`
+  display: flex;
+  justify-content: space-around;
+  gap: 1rem;
+  align-items: center;
+`;
+export default function FilterBar({ onChange, filters, reset }) {
   const [isFilterVisible, setIsFilterVisible] = useState(true);
 
   const toggleFilter = () => {
     setIsFilterVisible(!isFilterVisible);
   };
 
+  const { data: cities } = useSWR("/api/cities");
+  const { data: categories } = useSWR("/api/categories");
+
   return (
-    <div className="filter-container">
-      <button className="filter-toggle" onClick={toggleFilter}>
+    <StyledDiv>
+      <Button color="purple" onClick={toggleFilter}>
```

Comment:
Generally we prefer to pass a `variant` or `mode` prop to buttons, to better depict their role (e.g. `<Button variant="danger">` or `<Button variant="cancel">`), then we pick the associate color via styled components / CSS.

This means that all buttons that do the same thing...must have the same color!